ios/Classifier/VCPTaxonomy.m

```diff
@@ -108,6 +115,8 @@ - (NSDictionary *)leafScoresFromClassification:(MLMultiArray *)classification {
 
 - (NSArray *)inflateTopBranchFromClassification:(MLMultiArray *)classification {
     NSDictionary *scores = [self aggregateScores:classification];
+    // Log number of nodes in scores
+    NSLog(@"Number of nodes in scores: %lu", (unsigned long)scores.count);
```

Comment:
We could talk about if it makes sense to dynamically react to this number and increase or decrease depending on how many nodes are in here.

Reply:
We talked about this in a meeting and it makes more sense to first try with a taxonomic cutoff value that is relative to top leaf-node prediction score as in the web API.